### PR TITLE
[WIP] doc: improve documents style

### PR DESCRIPTION
**Pull request by AliMD**
I am proud to have the opportunity to review the codes of this great project.
This PR is WIP and need your review to continue.
Can we improve the code documents in this way?

**Before**
```ts
/* Single line document ... */
```
or
```ts
/** Start from first line!
 * bla bla bla ...
 * bla bla bla ...
 */
```
or
```ts
/**
* Not aligned comments
* asterisks not aligned with single space
* bla bla bla ...
*/
```

**After**
```ts
/**
 * Multi line description with aligned asterisks and @params and @example ...
 * bla bla bla ...
 *
 * @param {number} x - Bla bla ..
 * @return {number} bla bla ...
 *
 * @example
 *   await delay(1000);
 */
```

Will you allow me to continue?
Thank you in advance.

### CODE REVIEW

**Comment by nayeemrmn** [comment]
Please read https://deno.land/manual/contributing/style_guide#use-jsdoc-for-exported-symbols, this has already been decided against.

**Comment by AliMD** [comment]
@nayeemrmn Thanks, so can I continue?

**Comment by nayeemrmn** [comment]
@AliMD No, as I said this change is against the style guide for good reason: we have TS.

**Comment by AliMD** [comment]
@nayeemrmn I dont understand whats the diffrent with the style guide?
Its just about @param, @return, etc ...
I will remove it, but what about aligned asterisks and not start from first line, etc ...
Could you please more explain?
Thank you.

File: docs/contributing/style_guide.md [hunk 1]
```diff
@@ -245,12 +245,13 @@ And not
 
 Code examples should not utilise the triple-back tick (\`\`\`) notation or tags.
 They should just be marked by indentation, which requires a break before the
-block and 6 additional spaces for each line of the example. This is 4 more than
-the first column of the comment. For example:
+block and 6 additional spaces for each line of the example. For example:
 
 ```ts
-/** A straight forward comment and an example:
```
Comment (AliMD) [inline docs/contributing/style_guide.md:252]: Its not really butifull if comments started at the first line
Could you please let me to change it ?

File: docs/contributing/style_guide.md [hunk 2]
```diff
@@ -245,12 +245,13 @@ And not
...
+/**
+ * A straight forward comment and an example:
  *
+ * @example
```
Comment (AliMD) [inline docs/contributing/style_guide.md:254]: And could ypu please let me to add `@example` keyword for all examples?

**Comment by AliMD** [comment]
@nayeemrmn Im sorry for taking your time
I removed `@param` and `@return` lines, could you please review it again?
Thank you.

File: docs/contributing/style_guide.md (hunk 1 above)
Comment (AliMD) [reply docs/contributing/style_guide.md:252]: @nayeemrmn ok, as you wish, but please use the same format for all documets.
I mean if you like comments to start at the first line, its ok but please let me to change all the comments to be same code style.
Am I right?

**Comment by ry** [comment]
@AliMD we take minimalist choices where possible - including in amount of lines. If something can be said in a small sentence then it should. Closing without merge.

**Pull request closed without merging**